shuup_tests/admin/test_order_mass_actions.py

```diff
@@ -147,7 +147,7 @@ def test_mass_edit_orders4(rf, admin_user):
     assert response.status_code == 200
 
     if weasyprint:
-        assert response['Content-Disposition'] == 'attachment; filename=order_%s_delivery.pdf' % order.pk
+        assert response['Content-Disposition'] == 'attachment; filename=shipment_%s_delivery.pdf' % order.pk
```

Comment:
The `.travis.yml` should probably have `pip install weasyprint` to verify this...